.gitignore

```diff
@@ -0,0 +1,25 @@
+# eclipse
```

Comment:
This gitignore can be removed too since the one on the repo is more extensive and i think it covers everything listed here